lib/api/app_worker/element_handler.dart

```diff
@@ -241,17 +254,17 @@ class ElementRepresentation extends ElementIdentifier {
   }
 }
 
+enum ElementUpdateAction { update, remove, clear }
+
 class ElementUpdate {
-  final ElementRepresentation element;
-  final bool matches;
+  final ElementRepresentation? element;
+  final ElementUpdateAction action;
 
   const ElementUpdate({
     required this.element,
```

Comment:
make `this.element` not `required` (remove keyword)
This allows you to not have to pass `null` explicitly at line 67.